### ISSUE TITLE
Not compatible with Magento 2.3.4

### ISSUE THREAD

**Issue by FYvwotschel** [magento-hackathon/bestsellers-sorting-m2#2]
bin/magento setup:upgrade results in

```
Module 'MagentoHackathon_BestsellersSorting':
Installing data... The attribute set ID is incorrect. Verify the ID and try again.
```

**Comment by FYvwotschel** [comment]
Changing the $eavSetup->addAttributeToGroup(...) to

```
$eavSetup->addAttributeToGroup(
            \Magento\Catalog\Model\Product::ENTITY,
            $eavSetup->getDefaultAttributeSetId(\Magento\Catalog\Model\Product::ENTITY),
            $eavSetup->getDefaultAttributeGroupId(\Magento\Catalog\Model\Product::ENTITY),
            self::ATTRIBUTE_NAME
        );
```

fixed the problem for me.

**Comment by sprankhub** [comment]
Fixed with #3.
